### ISSUE TITLE
ThemePackage.getStylesheetsPath is deprecated.

### ISSUE THREAD

**Issue by johnmichel** [tyre/base16-eighties-dark#10]
Store package style sheets in the `styles/` directory instead of `stylesheets/` in the `base16-eighties-dark` package
```
ThemePackage.getStylesheetsPath (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:460:9)
ThemePackage.getStylesheetPaths (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:471:32)
ThemePackage.loadStylesheets (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:453:38)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/theme-package.js:47:19)
ThemePackage.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:163:15)
ThemePackage.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/theme-package.js:43:12)
```

The API call in use will be [deprecated on June 1](http://blog.atom.io/2015/05/01/removing-deprecated-apis.html).

**Comment by tyre** [comment]
Fixed with 54e9759

**Comment by johnmichel** [comment]
@tyre I think you need to run `apm publish` to get this update pushed to https://atom.io/themes.
